addons/google-analytics/README.md

```diff
@@ -20,9 +26,25 @@ module.exports = {
 }
 ```
 
-Then, set an environment variable
+## Configuration
+
+The analytics addon can be configured using story parameters with the `analytics` key.
+To configure globally, import `addParameters` from your app layer in your `preview.js` file.
 
+```js
+import { addParameters } from '@storybook/client-api';
+
+addParameters({
+  analytics: {
+    reactGAId: 'UA-000000-01',
+    reactGAOptions: {},
+  },
+});
```

Comment:
setConfig would make more sense.

Even more would defining this in `main.js`, and have a preset push it into the proper place where the runtime expects it. (such as setConfig)

Reply:
@ndelangen can you elaborate why `setConfig` would be better in this scenario ?

If I understand correctly, `setConfig` does not allow for local story overrides. I can think of future enhancements where we would want to allow users to specify different reactGA options or tracking codes for certain stories. Would that still work with a setConfig approach ?

Reply:
> allow users to specify different reactGA options or tracking codes for certain stories

I suspect there's no need for that.. Do you?

Reply:
I do, having different tracking codes is likely desired in cases where a medium to big UI library is published to different scoped NPM packages but hosted in a single repository.

for instance say we have a UI kit that is split up across the following NPM packages:

`@same_fancy_library/components`
`@some_fancy_library/utils`

This pattern is getting more and more popular, in those cases one could argue having separate storybook instances for each package, but from my experience having a single storybook instance, that hosts all stories is preferable from a end user UX perspective.

Furthermore I've seen instances where submodules of big component libraries are maintained by different teams who each have their own GA View / property that they want to observe.

Regardless if we want to optimise for this use-case, I think that the overhead of enabling this use-case is so small that it is likely worth exploring.

Reply:
@ndelangen WDYT ?

Reply:
I gave this some more thought, and given that storybook will get composition soon (https://github.com/storybookjs/storybook/pull/9210) my point is moot.

I'll look into updating this PR to use `setConfig`

Reply:
👍